middleware/.eslintrc

```diff
@@ -22,6 +22,7 @@
     ],
     "simple-import-sort/imports": "error",
     "simple-import-sort/exports": "error",
-    "@typescript-eslint/ban-ts-comment": "off"
+    "@typescript-eslint/ban-ts-comment": "off",
+    "@typescript-eslint/no-explicit-any": "off"
   }
```

Comment:
please turn it on before switching this PR from draft, we don't want to have `any[]` - please provide types for this data structures